corehq/apps/app_manager/views/modules.py

```diff
@@ -358,11 +360,21 @@ def _setup_case_property_builder(app):
 
 
 # Parent case selection in case list: get modules whose case type is the parent of the given module's case type
-def _get_modules_with_parent_case_type(app, module, case_property_builder, case_type_):
-        parent_types = case_property_builder.get_parent_types(case_type_)
-        modules = app.modules
-        parent_module_ids = [mod.unique_id for mod in modules
-                             if mod.case_type in parent_types]
+def _get_modules_with_parent_case_type(app, module, case_property_builder, case_type_, include_all=False):
```

Comment:
Stylistic, but I'd vote for two different, separately-named methods over one method with a boolean flag that changes the meaning of the method name ("parent" doesn't really apply if `include_all` is true).

Reply:
True. I have split these now.